automated-comments/go/hamming/comparing_bytes_disguised_as_runes.md

```diff
@@ -0,0 +1,5 @@
+Converting a `byte` to a `rune` works perfectly in the context of this exercise as there are only ASCII letters in DNA strands.
+ASCII characters consist of one `byte` each. As soon as non-ASCII characters get involved this will break as they consist of 
```

Comment:
```suggestion
ASCII characters consist of one `byte` each. As soon as non-ASCII characters get involved this method of comparison breaks as those characters consist of 
```

Just solidifying those references, as this sorta refers back to "this exercise" when "this method of comparing" (converting a byte to a rune and then doing the comparison) is meant.